lib/src/container/Container.tsx

```diff
@@ -41,53 +46,52 @@ const Container = styled.div<StyledProps>`
   float: ${({ float }) => float};
   z-index: ${({ zIndex }) => zIndex};
   box-shadow: ${({ boxShadow }) => boxShadow};
-  outline: ${({ outline }) => `${outline?.width} ${outline?.style} ${getCoreColorToken(outline?.color)}`};
-  outline-offset: ${({ outline }) => outline?.offset};
 
-  margin: ${({ margin }) => (typeof margin === "string" ? `${spaces[margin]}` : "")};
+  background-attachment: ${({ background }) => background?.attachment};
+  background-clip: ${({ background }) => background?.clip};
+  background-color: ${({ background }) => getCoreColorToken(background?.color)};
+  background-image: ${({ background }) => background?.image};
+  background-origin: ${({ background }) => background?.origin};
+  background-position: ${({ background }) => background?.position};
+  background-repeat: ${({ background }) => background?.repeat};
+  background-size: ${({ background }) => background?.size};
+
+  border-radius: ${({ borderRadius }) => borderRadius};
+  border-width: ${({ border }) => (border && "width" in border ? `${border?.width}` : "")};
+  border-style: ${({ border }) => (border && "style" in border ? `${border?.style}` : "")};
+  border-width: ${({ border }) => (border && "color" in border ? `${getCoreColorToken(border?.color)}` : "")};
+  ${({ border }) => {
+    if (border != null)
+      return "top" in border
+        ? getBorderStyles("top", border.top)
+        : "right" in border
+        ? getBorderStyles("right", border.right)
+        : "left" in border
+        ? getBorderStyles("left", border.left)
+        : "bottom" in border
+        ? getBorderStyles("bottom", border.bottom)
+        : "";
```

Comment:
There is an issue with this approach. It will only apply one side even if you have multiple sides.